src/server.nim

```diff
@@ -89,17 +111,17 @@ proc handle(client: Client; ev: Event) {.async.} =
     PlaylistAdd(url):
       checkPermission(janny)
       if "http" notin url:
-        client.send(Message("Invalid url"))
+        client.send(Message("server", "Invalid url"))
       else:
         playlist.add url
         broadcast(PlaylistAdd(url))
-        broadcast(Message(&"{client.name} added {url}"))
+        broadcast(Message("server", &"{client.name} added {url}"))
     PlaylistPlay(index):
       checkPermission(admin)
       if index > playlist.high:
-        client.send(Message("Index too high"))
+        client.send(Message("server", "Index too high"))
```

Comment:
There's a lot of these, maybe a util?
`client.sendEvent("Index too high")`
Consistent with the `showEvent` proc client-side